### PR TITLE
uar-150 fix missing no more to add

### PR DESCRIPTION
**Pull request by AndreKramer7654**
https://companieshouse.atlassian.net/browse/UAR-150 The no more to add button was missing in action

### Work checklist

- [ ] Tests added where applicable
- [ ] UI changes meet accessibility criteria

### Merge instructions

We are committed to keeping commit history clean, consistent and linear. To achieve this, this commit should be structured as follows:

```
<type>[optional scope]: <description>
```

and contain the following structural elements:

- fix: a commit that patches a bug in your codebase (this correlates with PATCH in semantic versioning),
- feat: a commit that introduces a new feature to the codebase (this correlates with MINOR in semantic versioning),
- BREAKING CHANGE: a commit that has a footer `BREAKING CHANGE:` introduces a breaking API change (correlating with MAJOR in semantic versioning). A BREAKING CHANGE can be part of commits of any type,
- types other than `fix:` and `feat:` are allowed, for example `build:`, `chore:`, `ci:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`, and others,
- footers other than `BREAKING CHANGE: <description>` may be provided.

### CODE REVIEW

**Review by olaych — COMMENTED** [review]
Another parameter is not need here. Changes only required on "beneficial-owner.html"

File: views/includes/page-templates/beneficial-owner.html [hunk 1]
```diff
@@ -38,11 +38,13 @@ <h1 class="govuk-heading-xl govuk-!-margin-0">{{ title }}</h1>
       <button class="govuk-button" data-module="govuk-button" id="add" formaction="{{ addButtonAction }}" >Add<span class="govuk-visually-hidden"> {{ whatToAdd }} </span></button>
 
       {% if (hasBeneficialOwners or hasManagingOfficers) %}
-        <h2 class="govuk-heading-l govuk-!-margin-0">What <span class="govuk-visually-hidden"> {{ hiddenTextValue }}</span> you have added so far:</h2>
-        <br>
-        {% include "includes/list/beneficial-owner.html" %}
-        {% include "includes/list/managing-officer.html" %}
-
+        
+        {% if not noLists %}
```
Comment (olaych) [inline views/includes/page-templates/beneficial-owner.html:42]: This is not needed. 

Adding "no more to add button" outside if block on line 45 is sufficient"

**Comment by AndreKramer7654** [comment]
It's fine to block out the lists for now. We will be re-visiting this later.

**Review by olaych — APPROVED** [review]
Not 100% sure about this approach but approving to unblock

**Review by jkayyala — APPROVED** [review]

File: views/includes/page-templates/beneficial-owner.html (hunk 1 above)
Comment (AndreKramer7654) [reply views/includes/page-templates/beneficial-owner.html:42]: We'll be working on expanding this page with the review BO/MO logic so it's wip.

**Pull request merged**